src/jdk.compiler/share/classes/com/sun/tools/javac/comp/Resolve.java

```diff
@@ -458,6 +462,19 @@ public boolean isAccessible(Env<AttrContext> env, Type site, Symbol sym, boolean
             return isAccessible(env, site, checkInner) && notOverriddenIn(site, sym);
         }
     }
+
+    private boolean privateMemberInPermitsClauseIfAllowed(Env<AttrContext> env, Symbol sym) {
+        if (env.info.isPermitsClause && ((JCClassDecl) env.tree).sym.outermostClass() == sym.owner.outermostClass()) {
+            if (allowPrivateMembersInPermitsClause) {
+                return true;
+            } else {
+                log.error(env.tree.pos(), Errors.PrivateMembersInPermitsClause);
```

Comment:
we don't issue a compiler error while checking if a symbol is accessible or not, it should be the caller's responsibility to decide if a compiler error should be issued or not as it is done in other cases

Reply:
thanks for the note, will keep in mind from now on